src/pages/TableValidationPage/TableValidationPage.module.css

```
@@ -0,0 +1,6 @@
+.wrapper {
+  textalign: "center";
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)